management/build.gradle

```diff
@@ -20,3 +20,9 @@ dependencies {
   compile project(':api'), project(':core'), project(':impl')
   compile group: 'org.terracotta.management', name: 'management-registry', version: parent.managementVersion
 }
+
+repositories {
+  mavenCentral()
+  maven { url "http://repo.terracotta.org/maven2" }
+  maven { url "http://snapshots.terracotta.org/" }
```

Comment:
This raises an interesting question. Ehcache 3 dependencies should be coming from Maven Central only - otherwise we break the implicit contract.
So if we start having to depend on the terracotta-management product in client libraries, it needs to be published to Sonatype snapshots and made ready for Central publication on release.
@mathieucarbou @anthonydahanne makes sense?